modules/@apostrophecms/schema/ui/apos/components/AposInputSelect.vue

```diff
@@ -37,13 +37,29 @@ export default {
       default: 'menu-down-icon'
     }
   },
+  data() {
+    return {
+      choices: []
+    };
+  },
+  mounted() {
+    if (!this.field.choices.find(choice => {
+      return !choice.value || choice.value === 'any';
+    })) {
+      this.choices.push({
```

Comment:
Should every select field have this? Maybe it should be `null`? This is to allow people to have no response or clear the field.